app/pb_kits/playbook/pb_icon_stat_value/icon_stat_value.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbIconStatValue
+    class IconStatValue
+      include Playbook::Props
+      include ActionView::Helpers::NumberHelper
+
+      partial "pb_icon_stat_value/icon_stat_value"
+
+      prop :icon, required: true
+
+      prop :size, type: Playbook::Props::Enum,
+                  values: %w[sm lg],
+                  default: "sm"
+      prop :variant, type: Playbook::Props::Enum,
+                     values: %w[default royal blue purple teal red yellow green],
+                     default: "default"
+
+      prop :orientation, type: Playbook::Props::Enum,
+                         values: %w[vertical horizontal],
+                         default: "horizontal"
+      prop :unit
+      prop :text
+      prop :value
+
```

Comment:
Can any of these "unit, text, value" support validation types like "icon, size, variant, orientation"?